### PR TITLE
updating copyright header

### PR DESCRIPTION
**Pull request by nnares**

### CODE REVIEW

File: management-common-resources-v2/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -12,7 +12,7 @@ org.eclipse.jdt.ui.keywordthis=false
...
...lock" name\="catchblock">// ${todo} Auto-generated catch block\r\n${exception_var}.printStackTrace();</template><template autoinsert\="true" context\="methodbody_context" deleted\="false" description\="Code in created function stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.methodbody" name\="methodbody">// ${todo} Auto-generated function stub\r\n${body_statement}</template><template autoinsert\="true" context\="constructorbody_context" deleted\="false" description\="Code in created constructor stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.constructorbody" name\="constructorbody">${body_statement}\r\n// ${todo} Auto-generated constructor stub</template><template autoinsert\="true" context\="getterbody_context" deleted\="false" description\="Code in created getters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.getterbody" name\="getterbody">return ${field};</template><template autoinsert\="true" context\="setterbody_context" deleted\="false" description\="Code in created setters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.setterbody" name\="setterbody">${field} \= ${param};</template></templates>
```
Comment (aurbroszniowski) [inline management-common-resources-v2/.settings/org.eclipse.jdt.ui.prefs:15]: the change about the IBM line misses a *

* Copyright Super iPaaS Integration LLC, an IBM Company 2024.\r\n

File: management-common-resources-v2/src/main/java/org/terracotta/management/application/DefaultApplicationV2.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
- * All content copyright (c) 2003-2012 Terracotta, Inc., except as may otherwise be noted in a separate copyright
- * notice. All rights reserved.
+ * Copyright Terracotta, Inc.
```
Comment (aurbroszniowski) [inline management-common-resources-v2/src/main/java/org/terracotta/management/application/DefaultApplicationV2.java:2]: copyright years must remain

File: management-common-resources-v2/src/main/java/org/terracotta/management/resource/services/AgentsResourceServiceImplV2.java [hunk 3]
```diff
@@ -1,4 +1,7 @@
-/* All content copyright (c) 2003-2012 Terracotta, Inc., except as may otherwise be noted in a separate copyright notice.  All rights reserved.*/
+/*
+ * Copyright Terracotta, Inc.
```
Comment (aurbroszniowski) [inline management-common-resources-v2/src/main/java/org/terracotta/management/resource/services/AgentsResourceServiceImplV2.java:2]: copyright years must remain

File: management-common-v2/.settings/org.eclipse.jdt.ui.prefs [hunk 4]
```diff
@@ -11,7 +11,7 @@ org.eclipse.jdt.ui.keywordthis=false
...
...lock" name\="catchblock">// ${todo} Auto-generated catch block\r\n${exception_var}.printStackTrace();</template><template autoinsert\="true" context\="methodbody_context" deleted\="false" description\="Code in created function stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.methodbody" name\="methodbody">// ${todo} Auto-generated function stub\r\n${body_statement}</template><template autoinsert\="true" context\="constructorbody_context" deleted\="false" description\="Code in created constructor stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.constructorbody" name\="constructorbody">${body_statement}\r\n// ${todo} Auto-generated constructor stub</template><template autoinsert\="true" context\="getterbody_context" deleted\="false" description\="Code in created getters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.getterbody" name\="getterbody">return ${field};</template><template autoinsert\="true" context\="setterbody_context" deleted\="false" description\="Code in created setters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.setterbody" name\="setterbody">${field} \= ${param};</template></templates>
```
Comment (aurbroszniowski) [inline management-common-v2/.settings/org.eclipse.jdt.ui.prefs:14]: missing * before Super IPAAS copyright

File: management-common-v2/src/main/java/org/terracotta/management/resource/AbstractEntityV2.java [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /*
- * All content copyright (c) 2003-2012 Terracotta, Inc., except as may otherwise be noted in a separate copyright
- * notice. All rights reserved.
+ * Copyright Terracotta, Inc.
```
Comment (aurbroszniowski) [inline management-common-v2/src/main/java/org/terracotta/management/resource/AbstractEntityV2.java:2]: copyright years must remain

**Review by aurbroszniowski — CHANGES_REQUESTED** [review]

File: management-common-resources-v2/.settings/org.eclipse.jdt.ui.prefs (hunk 1 above)
Comment (nnares) [reply management-common-resources-v2/.settings/org.eclipse.jdt.ui.prefs:15]: added '*' after \r\n and prior to Copyright Super iPaaS Integration LLC, an IBM Company 2024.\r\n

File: management-common-v2/.settings/org.eclipse.jdt.ui.prefs (hunk 4 above)
Comment (nnares) [reply management-common-v2/.settings/org.eclipse.jdt.ui.prefs]: added '*' after \r\n and prior to Copyright Super iPaaS Integration LLC, an IBM Company 2024.\r\n

File: management-common-v2/src/main/java/org/terracotta/management/resource/AbstractEntityV2.java (hunk 5 above)
Comment (nnares) [reply management-common-v2/src/main/java/org/terracotta/management/resource/AbstractEntityV2.java]: Restored the last copyright text with years 
added this over existing copyright - * Copyright Super iPaaS Integration LLC, an IBM Company 2024

File: management-common-resources-v2/src/main/java/org/terracotta/management/resource/services/AgentsResourceServiceImplV2.java (hunk 3 above)
Comment (nnares) [reply management-common-resources-v2/src/main/java/org/terracotta/management/resource/services/AgentsResourceServiceImplV2.java]: Restored the last copyright text with years 
added this over existing copyright - * Copyright Super iPaaS Integration LLC, an IBM Company 2024

File: management-common-resources-v2/src/main/java/org/terracotta/management/application/DefaultApplicationV2.java (hunk 2 above)
Comment (nnares) [reply management-common-resources-v2/src/main/java/org/terracotta/management/application/DefaultApplicationV2.java]: Restored the last copyright text with years 
added this over existing copyright - * Copyright Super iPaaS Integration LLC, an IBM Company 2024

**Pull request merged**